.github/ISSUE_TEMPLATE.md

```diff
@@ -11,14 +11,12 @@ For Bug report or feature requests, please fill out the relevant category below
 
 - Operating System:
   - <!-- OS and version (e.g. Windows 10, Ubuntu 16.04...) -->
```

Comment:
Can you keep the installation type? it makes it a little easier to gauge the technical-prowess of the asker (if from source, I generally expect a little more code-editing capabilities)

Reply:
That's part of the version, which is why I removed it.

from source, it will look like: `19dd3302d53edf77ec80a09e95fe44691946cfb1`
from binaries: `ros-dashing-navigation2	0.2.4-1bionic.20190911.163256`

Reply:
I can live with that